### PR TITLE
Robe/deprecate win2003

### PR DESCRIPTION
**Pull request by zentron**
# About
The changes in our support policy around deployment targets, and more concretely the deprecation of support for Windows Server 2003 in 2024.1

# Release
None specifically, but the sooner the better in providing ample warning and room for discussions.

# Blog image idea
.. not sure... dont mind if its something generic. Maybe something tied to the idea of old platforms being carried on our back as being old and broken?

# Prerequisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [guide for blog content basics](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content) and [writing educational content](https://www.octopus.design/932c0f1a9/p/137b06-educational-content---how-we-write-about-what-we-do).
- [ ] I've checked the content using Grammarly (Octopus has a corporate license) and the [Hemingway app](https://hemingwayapp.com/) to remove typos, grammatical errors, and passive voice. 
- [x] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content/t/817249) and none of those apply.
- [x] The images I've included follow the [rules for working with images](https://www.octopus.design/932c0f1a9/p/5061d7-working-with-images).
- [x] I've added alt text for all the images I've included (alt text describes an image to people unable to see it, 125 characters max)
- [ ] I've drafted social media posts for Twitter and LinkedIn and will post them for review in #topic-social-media.
- [ ] I've dropped a note in Slack in #topic-content-marketing to let Tegan Ali know my post is ready for review. (No need to fill out a content brief.)

## How to review this PR
- [ ] Standard writing grammer, style etc
- [ ] Are we happy with the message we are sending about our planned schedule ( @michaelnoonan )

## Publication date
no concerns

### CODE REVIEW

File: blog/2023-07/deprecating-win2003/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+isFeatured: false
+tags: 
+- Product
+---
+
+Supporting outdated operating systems come with a cost that gets more expensive over time. As a result Octopus Deploy will drop support for Microsoft Windows 2003 Targets and Workers from the `2024.1` release. 
+
+This post aims to explain why we have had to make this change, as well as what you can do if this affects you.
+
+## Why deprecate Windows Server 2003
```
Comment (michaelnoonan) [inline blog/2023-07/deprecating-win2003/index.md:19]: I think this section is jarring in its current location. Consider moving it down to the FAQ-like section for people who want to know the gnarly details of why we chose this path. That way, the post will focus on the customer impact first, then the explanation later.

**Review by michaelnoonan — APPROVED** [review]
I think this is good to go barring one suggestion.

File: blog/2023-07/deprecating-win2003/index.md (hunk 1 above)
Comment (zentron) [reply blog/2023-07/deprecating-win2003/index.md:19]: good idea

**Review by michaelnoonan — APPROVED** [review]

File: blog/2023-07/deprecating-win2003/index.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+Given the age, technical limitations, and customer impact of the Windows 2003 platform, we'll drop support in the first planned deprecation release in 2024.1.
+
+## Summary
+
+Although Microsoft dropped support for Windows Server 2003 several years ago, Octopus has continued to invest in ensuring customers who rely on this OS can deploy to it. Unfortunately, supporting outdated platforms has a cost for us and encourages bad practices.
+
+From the 2024.1 release onwards, Windows Server 2003 will become an unsupported platform for deployment targets or Workers.
+
+If there's anything in this post that concerns you, or you'd like further information, please get in touch with our[support team](mailto:support@octopus.com) or add a comment below.
```
Comment (TeganAli) [inline blog/2023-07/deprecating-win2003/index.md:96]: @zentron is support@octopus.com the correct address to use here, or should it be customersuccess@octopus.com ? (And does the appropriate team know this post is being published soon?)

File: blog/2023-08/deprecating-win2003/index.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
+---
+title: Dropping support for Windows Server 2003 machines
+description: Octopus Deploy will drop support for targets and Workers running Windows Server 2003 from the 2024.1 release.
+author: robert.erez@octopus.com
+visibility: public
+published: 2023-08-21-1400
+metaImage: img-blog-unsupporteddeploymenttargetincwindows-2023.png
+bannerImage: img-blog-unsupporteddeploymenttargetincwindows-2023.png
+bannerImageAlt: Silhouette of a woman wearing headphones with a mic in an Octopus branded tee holding a tablet and
```
Comment (TeganAli) [inline blog/2023-08/deprecating-win2003/index.md:9]:
```suggestion
bannerImageAlt: Silhouette of a woman wearing headphones with a mic in an Octopus branded tee holding a tablet.
```

**Pull request merged**